src/connectors/youtube-music.js

```diff
@@ -23,14 +23,14 @@ Connector.albumSelector = [
 	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/FEmusic_library_privately_owned_release_detailb_"]',
 ];
 
+const videoHasAlbum = () => !!Connector.getAlbum();
```

Comment:
Please don't use function expressions. We use function declarations everywhere to declare a function.